lib/ember-app.js

```diff
@@ -32,16 +29,38 @@ function EmberApp(options) {
   sandbox.run(appFile);
   debug("app file evaluated");
 
-  this.waitForBoot = function() {
-    return appBoot.promise;
-  };
+  var AppFactory = sandbox.require('~fast-boot/app-factory');
```

Comment:
I copied it from Stef :P I think it is just to make it less likely to collide with user codez; on the other hand, I already relied on a globa called `FastBoot` so...